.vib/keycloak/cypress/cypress.json

```
@@ -0,0 +1,9 @@
+{
+    "baseUrl": "http://localhost",
+    "pageLoadTimeout": 240000,
+    "defaultCommandTimeout":40000,
+    "env": {
+      "username": "user",
+      "password": "ComplicatedPassword1234!"
```

```suggestion
      "password": "ComplicatedPassword123!4"
```

Remember to update it in the `runtime_params`

---

Done on both places.